test/utils/integration-tests.sol

```diff
@@ -925,6 +939,29 @@ contract DGRegressionTestSetup is DGScenarioTestSetup {
 
         _setTimelock(_dgDeployedContracts.timelock);
 
+        console.log("Executing already submitted proposals...");
+        uint256 proposalsCount = _timelock.getProposalsCount();
+        _wait(_getAfterSubmitDelay());
```

Comment:
What if current DG state is VetoSignalling? In this case all proposals won't be submitted silently. I would suggest one of two ways: fail with the error "unexpected state" or process through this state.